### ISSUE TITLE
Trix turns $styling into class for no reason

### ISSUE THREAD

**Issue by stancl** [blade-ui-kit/blade-ui-kit#76]
The Trix component doesn't transform `$styling` at all, it simply does `class="{{ $styling }}"`

This is inconsistent with e.g. Pikaday, which has no `$styling` prop and simply passes attributes (including `class`) through.

Also, calling it *styling* is a bit inaccurate. It just goes into class. Not inline CSS. Classes can serve more purposes than just styling.

So I think the `$styling` prop on Trix should be renamed to `$class`.

**Comment by RomainMazB** [comment]
Hi @stancl ,
As far as I see it's not due to a bug but to a misunderstand, maybe the docs should be more concise or the parameter's name changed (it was not so obvious for me too when I read your issue).

Viewing [the source](https://github.com/blade-ui-kit/blade-ui-kit/blob/0.x/resources/views/components/editors/trix.blade.php#L3) :
```html
<div {{ $attributes }}>
    <input name="{{ $name }}" id="{{ $id }}" value="{{ old($name, $slot) }}" type="hidden">
    <trix-editor input="{{ $id }}" class="{{ $styling }}"></trix-editor>
</div>
```
and [the doc](https://blade-ui-kit.com/docs/0.x/trix) (where it talks about Styling), the styling parameter is just passed as class into the trix-editor tag where the basic class parameters is passed to the container div (into the `{{$attributes}}`).

So to be more clear: to style your trix-editor tag, you can't use directly the inline-styles: you'll need to create a class, or use Tailwind's ones.

Maybe another parameters to directly inject styles should be added but I'm not sure it would be useful...

**Comment by stancl** [comment]
Ah, I see. It's passed to a different element than the rest of attributes.

Makes sense. Though maybe it would be better to name it `$classes` or something, because classes aren't necessarily styling, and styling isn't necessarily classes.

**Comment by RomainMazB** [comment]
I agree with with you this can lead to confusion. I've faced in a personal project the same problem of "how should I name this second parameter which also contains classes?"

At the end I just named it something like "trixClass" or something like that.
